ui/app/templates/components/identity/entity-nav.hbs

```diff
@@ -1,7 +1,7 @@
 <PageHeader as |p|>
   <p.levelLeft>
     <h1 class="title is-3">
-      {{capitalize (pluralize this.identityType)}}
+      {{titleize (pluralize this.identityType)}}
```

Comment:
sometimes `identityType` is two words, so to play it safe using the titlelize helper which caps all words not just the first.